vortex-flatbuffers/src/generated/array.rs

```diff
@@ -93,7 +93,7 @@ impl<'a> flatbuffers::Verifiable for Version {
 
 impl flatbuffers::SimpleToVerifyInSlice for Version {}
 pub enum ArrayOffset {}
-#[derive(Copy, Clone, PartialEq, Eq)]
+#[derive(Copy, Clone, PartialEq)]
```

Comment:
I'm not sure why the generator stripped `Eq` this time, but I don't think it matters